### PR TITLE
add: products,cart,wishlist,forgot-pass page

### PR DESCRIPTION
**Pull request by thegeorgenikhil**
added
- forgot password page [Link](https://deploy-preview-2--floww-dot.netlify.app/src/auth/forgot-pass.html)
- cart page [Link](https://deploy-preview-2--floww-dot.netlify.app/src/pages/cart.html)
- wishlist page [Link](https://deploy-preview-2--floww-dot.netlify.app/src/pages/wishlist.html)
- product page [Link](https://deploy-preview-2--floww-dot.netlify.app/src/pages/products.html)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -94,6 +104,209 @@ body {
...
+}
+
+.cart-products-container {
+  background-color: var(--color-white);
+  padding: 1rem;
+  margin: 2rem 0;
+  border-radius: var(--border-sm);
+  box-shadow: var(--shadow-lg);
+}
+
```
Comment (Hnh013) [inline styles.css:278]: A suggestion: if feasible, try to make variable for certain values which are used very frequently , like 1rem

**Review by Hnh013 — APPROVED** [review]
added a suggestion , rest all is good code is clean !

File: styles.css (hunk 1 above)
Comment (thegeorgenikhil) [reply styles.css:278]: Will look into that thank you👍

**Review by MurtazaBagwala07 — APPROVED** [review]
Pages look nice

**Pull request merged**